### PR TITLE
Rewrite profile view

### LINKED ISSUE #342: Profile view becomes sluggish when filled with 1k+ works

### PR DESCRIPTION
**Pull request by RaitoBezarius**
- [x] Paginated.
- [x] Fast.
- [x] Use `{% static 'img/ratings.svg' %}` once.
- [x] Do not break other pages which depended on `rating.html` templates.
- [x] DPP-compatible.
- [x] More readable code.
- [x] Privacy-friendly (for private profiles).
- [x] Optimised (don't query stuff which is not going to be used).
- [x] Anonymous rating-compatible.
- [x] Marked areas which should be soon removed (deprecated, like neighbour system).

Fixes #342 and #363.

### CODE REVIEW

File: mangaki/mangaki/templates/profile_work.html [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+            {% elif category == 'album' %}
+                <a href="{% url 'work-list' 'album' %}">Albums</a>
+                {% if seen %}écoutés {% else %} à écouter (ou non) {% endif %}
+            {% else %}
+                <a href="{% url 'work-list' category %}">{{ category }}</a>
+                {% if seen %}consommés {% else %} à consommer (ou non) {% endif %}
+            {% endif %}
+        </h1>
+        <p class="well">✨🍰✨ Trier selon Mangaki !
+            <a href="?algo=svd">mainstream</a> /
```
Comment (RaitoBezarius) [inline mangaki/mangaki/templates/profile_work.html]: Note to myself: should add `page`.

File: mangaki/mangaki/templates/profile_work.html [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+                </li>
+                {% for page in ratings.paginator.page_range %}
+                    {% if page <= ratings.number|add:3 and page >= ratings.number|add:-3 %}
+                        <li
+                                {% if page == ratings.number %}class="active" {% endif %}>
+                            <a
+                                    {% if page == ratings.number %}
+                                        href="#"
+                                    {% else %}
+                                        href="?page={{ page }}"
```
Comment (RaitoBezarius) [inline mangaki/mangaki/templates/profile_work.html]: Note to myself: should add `algo`.

File: mangaki/mangaki/views.py [hunk 3]
```diff
@@ -67,6 +74,55 @@
...
+        {
+            'name': "dislike",
+            'title': "Je n'aime pas"
+        },
+        {
+            'name': "dontknow",
+            'title': "Je ne connais pas"
+        }
+    ],
+    'endpoint': '/vote_dpp/'
```
Comment (RaitoBezarius) [inline mangaki/mangaki/views.py]: Note to myself: should use `reverse` I guess.

File: mangaki/mangaki/templates/profile_work.html [hunk 4]
```diff
@@ -0,0 +1,74 @@
+<div class="col-md-12">
+    <h1>
+        {% if meta.category == 'anime' %}
+            <a href="{% url 'work-list' 'anime' %}">Anime</a>
+            {% if seen %}vus {% else %} à voir (ou non) {% endif %}
```
Comment (jilljenn) [inline mangaki/mangaki/templates/profile_work.html]: Plz fix!

File: mangaki/mangaki/templates/profile.html [hunk 5]
```diff
@@ -76,138 +77,74 @@
...
-                    href="{% url 'profile' username %}?category=recommendation">Liste de recommandations
-                de {{ username }} <span class="badge alert-info">{{ reco_count }}</span></a></li>
-            {% if best_neighbors_list %}
-                <li role="presentation"{% if category == 'neighbor' %} class="active"{% endif %}><a
-                        href="{% url 'profile' username %}?category=neighbor">Les {{ neighbor_count }} plus proches
-                    voisins de {{ username }}</a></li>
-            {% endif %}
+
+            <li role="presentation"{% if meta.category == 'anime' and meta.seen %} class="active"{% endif %}><a
+                    href="{% url 'profile' profile.username 'anime' 'seen' %}">Liste d'<em>anime</em> vus de {{ profile.username }} <span
```
Comment (jilljenn) [inline mangaki/mangaki/templates/profile.html]: Anime vus*

**Comment by jilljenn** [comment]
Plz fix number of `reco_count`.

Also, it should not be computed on private profiles (but I assume it is already the case).

File: mangaki/mangaki/templates/profile.html [hunk 6]
```diff
@@ -4,29 +4,30 @@
...
+                    <img src="/static/img/chiro.png" alt="Avatar de {{ profile.username }}" id="avatar"/>
+{#                {% endif %}#}
             </div>
         {% endif %}
 
         <div class="col-xs-4 col-lg-6">
-            {% if not is_anonymous %}
-                <h1>{{ username }}</h1>
+            {% if not meta.is_anonymous %}
+                <h1>{{ profile.username }} {% if meta.is_me %} (vous!) {% endif %}</h1>
```
Comment (jilljenn) [inline mangaki/mangaki/templates/profile.html]: No `vous` please! En plus faudrait une espace insécable.

File: mangaki/mangaki/templates/profile_work.html [hunk 7]
```diff
@@ -0,0 +1,74 @@
...
+        {% elif meta.category == 'manga' %}
+            <a href="{% url 'work-list' 'manga' %}">Mangas</a>
+            {% if seen %}lus {% else %} à lire (ou non) {% endif %}
+        {% elif meta.category == 'album' %}
+            <a href="{% url 'work-list' 'album' %}">Albums</a>
+            {% if seen %}écoutés {% else %} à écouter (ou non) {% endif %}
+        {% else %}
+            <a href="{% url 'work-list' meta.category %}">{{ meta.category }}</a>
+            {% if seen %}consommés {% else %} à consommer (ou non) {% endif %}
```
Comment (jilljenn) [inline mangaki/mangaki/templates/profile_work.html]: Seen should not be considered for `recommendations`. Also it should be written `Recommandations` in the French version, not `recommendation` (check the typography, also).

File: mangaki/mangaki/templates/profile.html [hunk 8]
```diff
@@ -51,7 +52,7 @@
             {% endif %}
         </div>
         <div class="col-xs-6 col-lg-4">
-            {% if request.user.username == username %}
+            {% if meta.is_me and not meta.is_anonymous %}
                 <br/>
                 Je participe au data challenge de Kyoto :
```
Comment (jilljenn) [inline mangaki/mangaki/templates/profile.html]: Plz add unbreakable space.

**Comment by jilljenn** [comment]
In manga we get anime!

File: mangaki/mangaki/templates/profile.html [hunk 9]
```diff
@@ -76,138 +77,78 @@
...
+
+            <li role="presentation"{% if meta.category == 'recommendation' %} class="active"{% endif %}><a
+                    href="{% url 'profile' profile.username 'recommendation' %}?category=recommendation">Recommandations
+                de {{ profile.username }} <span class="badge alert-info">{{ profile.reco_count }}</span></a></li>
+        </ul>
+    {% elif meta.is_anonymous or meta.is_me %}
+        <ul class="nav nav-tabs">
+
+            <li role="presentation"{% if meta.category == 'anime' and meta.seen %} class="active"{% endif %}><a
+                    href="{% url 'my-profile' 'anime' 'seen' %}"><em>Anime</em> que j'ai vu <span
```
Comment (jilljenn) [inline mangaki/mangaki/templates/profile.html]: (anime) vus*
(mangas) lus*

Tu conserves l'italique ?

C'est mon dernier commentaire, après tu peux merge :)

Comment (RaitoBezarius) [inline mangaki/mangaki/templates/profile.html]: Yes, if we want to change this, it's quite easy.

**Pull request merged**